libs/sdk-py/langgraph_sdk/client.py

```diff
@@ -412,8 +412,7 @@ def stream(
         interrupt_after: Optional[list[str]] = None,
         feedback_keys: Optional[list[str]] = None,
         multitask_strategy: Optional[MultitaskStrategy] = None,
-    ) -> AsyncIterator[StreamPart]:
-        ...
+    ) -> AsyncIterator[StreamPart]: ...
```

Comment:
This is causing the lint error. The ellipse (`...`) should be on a new line. Undo this change and all of the other instances and you should be good to go!
```python
) -> AsyncIterator[StreamPart]:
    ...
```